### ISSUE TITLE
Bug: single integer values interpreted as HTTP status codes

### ISSUE THREAD

**Issue by bjrnt** [juliangruber/multilevel-http#13]
Hi,

When trying to receive a simple single integer value the value is instead interpreted as the status code of the response. 

Here's code to duplicate the error:

``` js
var levelup = require('level'),
multilevelHttp = require('multilevel-http'),
http = require('http');

var db = levelup('../myDb', {valueEncoding: 'json'});

db.put('lol', 1);

var app = multilevelHttp.server(db);
var server = http.createServer(app);
server.listen(1337);
```

The resulting behavior can then be found with curl:

``` bash
$ curl --verbose localhost:1337/data/lol
Hostname was NOT found in DNS cache
  Trying 127.0.0.1...
Connected to localhost (127.0.0.1) port 1337 (#0)
GET /data/lol HTTP/1.1
User-Agent: curl/7.35.0
Host: localhost:1337
Accept: */*

**HTTP/1.1 1 unknown**
X-Powered-By: Express
Content-Type: text/plain
Date: Mon, 02 Jun 2014 09:08:20 GMT
Connection: keep-alive
Transfer-Encoding: chunked

* Connection #0 to host localhost left intact
```

**Comment by juliangruber** [comment]
can you make a pull request with this code as a failing test case?

**Comment by bjrnt** [comment]
I was in the process of adding this to the tests, but I noticed that everything works as expected if you don't supply `{valueEncoding: 'json'}` in the initial levelup call, which isn't done when the db is initialized internally within this library. 

Tried this with and without the `valueEncoding` parameter:

```
var db = levelup('../myDb', {valueEncoding: 'json'});
db.put('lol', 1);
db.get('lol', function(err, val) {
    console.log('%s %s', val, typeof val);
 });
```

Which results were:

```
1 string (without encoding)
1 number (with encoding)
```

Since `res.send(123)` will interpret the int as the response code it explains the behavior I saw. I can submit a pull request with a fix, if you'd like. It would just be a special case when the returned value from the db is a number.
